kubernetes/README.md

```diff
@@ -0,0 +1,207 @@
+# torchelastic on Kubernetes
```

Comment:
Dear @Jeffwan - First of all, thank you for this amazing contribution to `torchelastic`! I have added a few minor comments for your consideration down below.